<launch>

  <node name="uav_commander" pkg="navigation" type="position_controller" ns="uav" output="screen">
    <rosparam command="load" file="$(find navigation)/config/pid_auto_params.yaml" />
  </node>

  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure"/>
</launch>
